### DISCUSSION TITLE
Delete button size

### DISCUSSION THREAD

**Discussion by bandaid** [sienori/Tab-Session-Manager#743] [Ideas]
Larger screens 4K plus... smaller windows and dialogs. 

Please update

**_Original dialog_**. Icons are too small and present all sorts of problems. Previous ticket#[741](https://github.com/sienori/Tab-Session-Manager/discussions/741) requesting to add confirmation dialog prior to the opening session. Also, no need for the 'Open' icon and dialog because when you click on the list to the left it opens anyway. It's just user-friendly than being cumbersome. 

![2021-04-12 18_18_44-](https://user-images.githubusercontent.com/8212255/114477577-eb2ac280-9beb-11eb-9115-d7924ffd787b.png)
.
.
.
.
### User should have a choice in Tab Session Manager 'Settings' for single or double width.
.
.
.
.
**New dialog with single width delete scaled to list item**

![single_size](https://user-images.githubusercontent.com/8212255/114477588-ef56e000-9beb-11eb-9cf2-be1ea8608d2a.png)
.
.
.
.
**New dialog with double width delete scaled to list item**

![double_size](https://user-images.githubusercontent.com/8212255/114477604-fa117500-9beb-11eb-824f-32cb8baa986b.png)

**Comment by sienori** [comment]
I improved it. https://github.com/sienori/Tab-Session-Manager/issues/744
